src/springfield/entity.py

```diff
@@ -105,9 +107,11 @@ def jsonify(self):
         Return a dictionary suitable for JSON encoding.
         """
         data = {}
-        for key, val in self.__values__.iteritems():
```

Comment:
Given our example above with `TestUser` the problems with `jsonify` should be more obvious. The difference is that `flatten` is usually used to put data in the database while `jsonify` is used to send data to the client. 

Here's an example of what would happen if defaults are included in `jsonify`:

1. User `Bob Test` is created as above with `id: AAAA`
2. `/api/users/AAAA` returns all of Bob's details
3. `/api/users` returns only the `id` and `name` of all users by doing `SELECT id, user FROM user`
4. Since only `id` and `user` is returned from the DB, `UserEntity` is only sparsely populated. However because of defaults, what we return for the HTTP response is `{"users": {"id": "AAAA", "name": "Bob Test", "enabled": True, "created": "2016-04-06T00:00:00Z"}}`.
5. There is now an unexpected `created` and `enabled` value in the response. These values are from the defaults so are completely invalid. This will confuse the HTTP client greatly.